doc/user/olm-catalog/generating-a-csv.md

```diff
@@ -93,6 +94,7 @@ Required:
 * `spec.displayName` _(user)_ : a name to display for the Operator in Operator Hub.
 * `spec.keywords` _(user)_ : a list of keywords describing the Operator.
 * `spec.maintainers` _(user)_ : a list of human or organizational entities maintaining the Operator, with a `name` and `email`.
+* `spec.icon` _(user)_ : a Base64 encoded image. E.g `cat image.png | base64` or `base64 -w 0 image.jpg`. 
```

Comment:
If we do go forward with this PR, we should be explicit about which image types are supported.

But, let's hold off on further changes until @estroz weighs in on [my previous question](https://github.com/operator-framework/operator-sdk/pull/2095#pullrequestreview-309890503) so we don't use CI resources unnecessarily.